demisto_sdk/commands/validate/tests/RM_validators_test.py

```diff
@@ -1126,3 +1126,40 @@ def test_invalid_short_file():
         not_to_short_readme_validator.obtain_invalid_content_items([test_pack])
     )
     assert result[0].message == short_readme_error
+
+
+def test_ImagePathIntegrationValidator_content_assets():
+    """
+        Given
+        content_items.
+        - Pack with:
+            1. invalid readme that contains absolute path.
+            2. description contains relative path that saved not under dec_files.
```

Comment:
```suggestion
            1. invalid readme that contains absolute path.
            2. description contains relative path that saved not under dec_files.
```

Let's list the exact files that are given and the ones that are invalid - will be better for documenting the expected behavior.